src/test/java/com/rackspace/ceres/app/services/DownsampleProcessorTest.java

```diff
@@ -104,14 +105,14 @@ void partitionsToProcessConfig() {
         .containsExactly(0, 1, 4, 5, 6, 7);
   }
 
+  private List<ScheduledFuture<?>> scheduled;
+
   @Test
   void setupSchedulers() {
-    Assertions.fail("TODO");
-  }
+    downsampleProcessor.setupSchedulers();
 
-  @Test
-  void processDownsampleSet() {
-    Assertions.fail("TODO");
+    //using test profile so scheduled must be empty as setupSchedulers wont run for test profile
+    assertThat(scheduled).isNull();
```

Comment:
I feel like this is worse than having the TODO there. Because with the TODO we know we need to test it still. This isn't actually testing anything.

Reply:
at first, I thought of removing that test case as Downsample scheduling has been disabled for test profile as per code. 

https://github.com/racker/ceres/blob/0955ca5dc156ccd10654b74836a459050c399773/src/main/java/com/rackspace/ceres/app/services/DownsampleProcessor.java#L96

Reply:
We could enable downsample scheduling for the testing profile. But only have very short downsample periods. I dont think it relevant to have multiple different tiers but testing raw data and then testing a 10 second period.